### DISCUSSION TITLE
Installing on Ubuntu and got "EACCES: permission denied".

### DISCUSSION THREAD

**Discussion by gjfLeo** [requarks/wiki#6924] [Help / Questions] [answered]
I followed https://docs.requarks.io/install/ubuntu on a new Ubuntu server. But the 'wiki' container cannot start (which keeps restarting). I used `docker logs wiki` and got this message:
```
DB_PASS_FILE is defined. Will use secret from file.
>>> Failed to read Docker Secret File using path defined in DB_PASS_FILE env variable!
EACCES: permission denied, open '/etc/wiki/.db-secret'
```

I have the pass file in '/etc/wiki/.db-secret' indeed, and the 'db' container seems work well.

Any assistance would be helpful and my apologies if I've misread something or missed some documentation somewhere.

**Comment by fnafox** [comment, marked as answer]
wiki user inside the docker container doesn't have read permission for `/etc/wiki/.db-secret`
adding the others read permission to the file will resolve that issue `chmod o+r .db-secret`
alternatively change the owner of the file to user id 1000 which the wiki runs under `chown 1000 .db-secret`
finally, but least recommended is to run the wiki container as root by adding the appropriate user to the docker run command `docker run -u=root`

**Reply by gjfLeo** [reply]
Thank you very much!
